### ISSUE TITLE
"plantings" violates foreign key constraint

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#471]
On pr/dev plantings silently get removed:

```
[2023-06-18T07:03:37Z DEBUG tokio_postgres::query] executing statement s20 with parameters: [ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 2950, array_oid: 2951 })), Some([69, 59, 122, 246, 96, 50, 75, 30, 138, 197, 192, 254, 125, 98, 249, 31])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 23, array_oid: 1007 })), Some([0, 0, 0, 2])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 23, array_oid: 1007 })), Some([0, 0, 7, 95])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 23, array_oid: 1007 })), Some([0, 0, 0, 78])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 23, array_oid: 1007 })), Some([0, 0, 0, 202])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 23, array_oid: 1007 })), Some([0, 0, 0, 100])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 23, array_oid: 1007 })), Some([0, 0, 0, 100])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 700, array_oid: 1021 })), Some([0, 0, 0, 0])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 700, array_oid: 1021 })), Some([63, 128, 0, 0])), ToSqlHelper(PgTypeMetadata(Ok(InnerPgTypeMetadata { oid: 700, array_oid: 1021 })), Some([63, 128, 0, 0]))]
[2023-06-18T07:03:37Z ERROR backend::error] Error executing diesel SQL query: insert or update on table "plantings" violates foreign key constraint "plantings_layer_id_fkey"
[2023-06-18T07:03:37Z DEBUG actix_web::middleware::logger] Error in response: ServiceError { status_code: 500, reason: "insert or update on table \"plantings\" violates foreign key constraint \"plantings_layer_id_fkey\"" }
[2023-06-18T07:03:37Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /api/maps/1/layers/plants/plantings HTTP/1.1" 500 95 "https://dev.permaplant.net/map" "Mozilla/5.0 (Android 6.0.1; Mobile; rv:109.0) Gecko/114.0 Firefox/114.0" 0.001993
```

**Comment by markus2330** [comment]
Applied workaround on dev.permaplant.net: `ALTER TABLE plantings DROP CONSTRAINT plantings_layer_id_fkey;`

**Comment by markus2330** [comment]
Fixed via #488
